hack/util.sh

```diff
@@ -237,6 +237,26 @@ function tryuntil {
 	done
 }
 
+# tryuntil_failure loops, retrying an action until it fails or times out after 90 seconds.
+function tryuntil_failure {
```

Comment:
There is no way to get the quotation nesting correct that I know of when you have three or four layers of it, which is what you need when using that construct.